### ISSUE TITLE
Issue in ShapeShader (need float constants for GLES)

### ISSUE THREAD

**Issue by Sungray** [matiasah/shadows#17]
```
Shadows.ShapeShader = love.graphics.newShader [[
	
	vec4 effect(vec4 Color, Image Texture, vec2 textureCoord, vec2 pixelCoord) {
		
		vec4 pixel = Texel(Texture, textureCoord);
		
		if ( pixel.a > 0 ) {
			
			if ( pixel.r > 0 || pixel.g > 0 || pixel.b > 0 ) {
				
				return Color;
				
			}
			
		}
		
		return vec4(0.0, 0.0, 0.0, 0.0);
		
	}
	
]]
```

0 should be 0.0 :

```
Shadows.ShapeShader = love.graphics.newShader [[
	
	vec4 effect(vec4 Color, Image Texture, vec2 textureCoord, vec2 pixelCoord) {
		
		vec4 pixel = Texel(Texture, textureCoord);
		
		if ( pixel.a > 0.0 ) {
			
			if ( pixel.r > 0.0 || pixel.g > 0.0 || pixel.b > 0.0 ) {
				
				return Color;
				
			}
			
		}
		
		return vec4(0.0, 0.0, 0.0, 0.0);
		
	}
	
]]
```

**Comment by matiasah** [comment]
You could perhaps open a pull request next time, but thank you anyway!

**Comment by matiasah** [comment]
Fixed in ad9e0a0

**Comment by Sungray** [comment]
Will do, not used to GitHub ! :)
